source/operations/install-deploy-manage/deploy-minio-tenant-helm.rst

```diff
@@ -76,64 +89,113 @@ You can modify the Operator deployment after installation.
+   
+   The following fields share the ``tenant.pools[0]`` prefix and control the number of servers, volumes per server, and storage class of all pods deployed in the Tenant:
+   
+   .. list-table::
+      :header-rows: 1
+      :widths: 30 70
 
-      helm install \
-        --namespace MINIO_TENANT_NAMESPACE \
-        --create-namespace \ 
-        MINIO_TENANT_NAME minio-operator/tenant
+      * - Field
+        - Description
 
-   For details on the options available in the MinIO Tenant ``values.yaml``, see :ref:`minio-tenant-chart-values`.
+      * - ``servers`` 
+        - The number of MinIO pods to deploy in the Server Pool.
+      * - ``volumesPerServer`` 
+        - The number of persistent volumes to attach to each MinIO pod (``servers``).
+          The Operator generates ``volumesPerServer x servers`` Persistant Volume Claims for the Tenant.
+      * - ``storageClassName`` 
+        - The Kubernetes storage class to associate with the generated Persistent Volume Claims.
 
+          If no storage class exists matching the specified value *or* if the specified storage class cannot meet the requested number of PVCs or storage capacity, the Tenant may fail to start.
 
-#. Validate the Tenant installation
+      * - ``size``
+        - The amount of storage to request for each generated PVC.
 
-   Check the contents of the specified namespace to ensure all pods and services have started successfully.
+#. Configure Tenant Affinity or Anti-Affinity
 
-   .. code-block:: shell
-      :class: copyable
+   The Tenant Chart supports the following Kubernetes Selector, Affinity and Anti-Affinity configurations:
 
-      kubectl get all -n MINIO_TENANT_NAMESPACE
+   - Node Selector (``tenant.nodeSelector``)
+   - Node/Pod Affinity or Anti-Affinity (``spec.pools[n].affinity``)
 
-   All pods and services should have a READY state before proceeding.
+   MinIO recommends configuring Tenants with Pod Anti-Affinity to ensure that the Kubernetes schedule does not schedule multiple pods on the same worker node.
 
-#. Expose the Tenant Console port
+   If you have specific worker nodes on which you want to deploy the tenant, pass those node labels or filters to the ``nodeSelector`` or ``affinity`` field to constrain the scheduler to place pods on those nodes.
 
-   Use ``kubectl port-forward`` to temporarily forward traffic from the MinIO pod to your local machine:
+#. Configure Network Encryption
+
+   The MinIO Tenant CRD provides the following fields from which you can configure tenant TLS network encryption:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 30 70
+
+      * - Field
+        - Description
+
+      * - ``tenant.certificate.requestAutoCert``
+        - Enables or disables MinIO :ref:`automatic TLS certificate generation <minio-tls>`
+
+      * - ``tenant.certificate.certConfig``
+        - Controls the settings for :ref:`automatic TLS <minio-tls>`.
+          Requires ``spec.requestAutoCert: true``
```

Comment:
"Controls" is a bit vague, is it worth mentioning the type of accepted values here? There is more info in the CRD, which is reasonable to presume the reader is looking at.

Reply:
Also `spec.requestAutoCert` can be `true`, `false`, or simply not present. It is not a required field.

Reply:
@allanrogerr iirc if `requestAutoCert`` is ``false`` then changes to this field do nothing, right?

Indeed omitting `requestAutoCert` defaults to `true` behavior so I could reverse this

```suggestion
      * - ``tenant.certificate.certConfig``
        - Controls the settings for :ref:`automatic TLS <minio-tls>`.
          Only has affect if ``spec.requestAutoCert`` is not false.
```

Reply:
The easiest path here is to just state 'controls he settings...if enabled` and let the user infer that disabling one thing disables the othe.r